main.tf

```diff
@@ -84,9 +84,14 @@ resource "aws_autoscaling_group" "this" {
   health_check_grace_period = 30
   vpc_zone_identifier       = var.subnet_ids
 
-  tag {
-    key                 = "Name"
-    value               = var.name
-    propagate_at_launch = true
-  }
+  tags = concat(
```

Comment:
Sorry for being picky, but I think we missed out, that `tags` is deprecated.

> [tags](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/autoscaling_group#tags) (Optional, Deprecated use tag instead) Set of maps containing resource tags. Conflicts with tag. See [Tags](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/autoscaling_group#tag-and-tags) below for more details.

I think we have to rewrite it with `dynamic` blocks.

Reply:
https://www.terraform.io/language/expressions/dynamic-blocks

Reply:
Hello @frank-bee, would you be willing to allow us to help with solving this change request?
I'm happy to push a commit that I have locally for addressing it, but before I can do so, you'd need to allow us to change your PR, as explained here: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork

Reply:
In case that's something you'd rather not have us do, you can mirror my changes from this [branch](https://github.com/Flaconi/terraform-aws-bastion-ssm-iam/tree/frank-bee-patch-1).